lib/features/keyboard/Keyboard.js

```diff
@@ -125,10 +125,17 @@ Keyboard.prototype._isEventIgnored = function(event) {
 /**
  * Bind keyboard events to the given DOM node.
  *
+ * @overlord
+ * @deprecated
```

Comment:
```suggestion
 * @deprecated Since version 15.0.0, do not specify the `node` parameter anymore.
```

Perhaps we can have more informative deprecation message.

Reply:
![capture HEGYmm_optimized](https://github.com/user-attachments/assets/4b582d6d-a8c5-4c6a-982e-93a52c40c242)